loader.go

```diff
@@ -27,6 +27,7 @@ type Loader struct {
 // NewLoader constructs a new Loader struct
 func NewLoader() *Loader {
 	l := log.Stderr()
+	l.SetLevel(log.Verbose)
```

Comment:
OK for now, but should figure out how to set verbosity and stream trace messages to client.  See #7.